reactive/kubernetes_worker.py

```diff
@@ -128,6 +133,9 @@ def upgrade_charm():
         set_state('kubernetes-master-worker-base.registry.configured')
         remove_state('kubernetes-worker.registry.configured')
 
+    if is_flag_set('nrpe-external-master.available'):
+        update_nrpe_config()
```

Comment:
i don't think this is necessary.  if we have nrpe, the `update_nrpe_config` will fire on its own.  doesn't seem like we need an explicit callout in `upgrade_charm`.

Reply:
i don't think so, but i'll defer to @johnsca -- pretty sure the reactive loop will gather up all handlers that are able to be run, and run them.

@johnsca, the question for you is whether or not the `upgrade-charm` hook handler needs to explicitly call funcs.  My understanding is that handlers will be ack'd and run based on the flags regardless of any 'hook' -- that is, if something in `upgrade-charm` twiddles a flag, reactive will queue it to run.

Reply:
The update_nrpe_charm function has a when_any:
```
@when_any('config.changed.nagios_context',
          'config.changed.nagios_servicegroups',
          'nrpe-external-master.reconfigure')
```

If none of those are set, we won't run this handler, right?
```
@when('kubernetes-worker.config.created')
@when('nrpe-external-master.available')
@when('kube-api-endpoint.available')
@when('kube-control.auth.available')
@when_any('config.changed.nagios_context',
          'config.changed.nagios_servicegroups',
          'nrpe-external-master.reconfigure')
def update_nrpe_config():
```
```

Reply:
handled in #42